static/js/notifications.js

```diff
@@ -701,14 +696,20 @@ export function reify_message_id(opts) {
 }
 
 export function register_click_handlers() {
-    $("#out-of-view-notification").on("click", ".compose_notification_narrow_by_topic", (e) => {
-        const message_id = $(e.currentTarget).data("message-id");
-        narrow.by_topic(message_id, {trigger: "compose_notification"});
-        e.stopPropagation();
-        e.preventDefault();
-    });
-    $("#out-of-view-notification").on("click", ".out-of-view-notification-close", (e) => {
-        clear_compose_notifications();
+    $("#above_composebox_banners").on(
+        "click",
+        ".narrow_to_recipient .above_composebox_banner_action_link",
+        (e) => {
+            const message_id = $(e.target).data("message-id");
+            narrow.by_topic(message_id, {trigger: "compose_notification"});
+            e.stopPropagation();
+            e.preventDefault();
+        },
+    );
+
+    $("#above_composebox_banners").on("click", ".above_composebox_banner_close_button", (e) => {
+        const $target = $(e.target).parents(".above_composebox_banner");
+        $target.remove();
```

Comment:
This removes just that banner, instead of calling `clear_compose_notifications` which removes all banners of this type. Most of the time (all of the time?) that would do the same thing.

Reply:
I think this is the correct way to define an `x` element, yeah.